modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase

```diff
@@ -329,4 +329,74 @@ definition {
 			webContentText = "Este é um campo de texto");
 	}
 
+	@description = "This is a test for LRQA-66427. This test check it's not possible to translate a field with Localizable property set as Off in a simple field and a Field Group into a Fieldset"
+	@priority = "5"
+	test ValidateNotTranslatableOnFieldset {
+		property portal.acceptance = "true";
```

Comment:
@thaisfrota this is not an acceptance teste, please remove this property, and since the test is currently failing, add the quarantine flag.
Thanks

Reply:
Done.